.github/workflows/release-python.yml

```diff
@@ -48,9 +48,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           allow-prereleases: true
       - name: Download a previously created wheel
-        uses: actions/download-artifact@v2
+        uses: actions/download-artifact@v4
         with:
-          name: python_bsonjs-3.8-${{ matrix.os }}-wheel
+          name: wheel-${{ github.run_id }}
```

Comment:
These look alright?

Reply:
I copied the way we handled the update from the older versions of the upload/download actions in `mongo-arrow`.